draft-ietf-trans-rfc6962-bis.md

```diff
@@ -1447,9 +1449,9 @@ as returned by `get-sth` in {{get-sth}}.
 
 The `start` parameter MUST be less than or equal to the `end` parameter.
 
-The `chain` field in the `submission` output parameter MUST include the trust
-anchor that the log used to verify the submission, even if it was omitted in the
-original submission.
+The `chain` field in the `submitted_entry` output parameter MUST include the
```

Comment:
s/in the/in each/

(There's a separate `submitted_entry` output parameter for each element in the `entries` output parameter).